### PR TITLE
{lib,tools,vis}[foss/2023a] imgaug v0.4.1, OpenCV v4.8.1, CUDA 12.1.1 contrib

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen3

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/pdf2docx/pdf2docx-0.5.8-foss-2023a-CUDA-12.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+easyblock = "PythonBundle"
+
+name = 'pdf2docx'
```
Comment (lexming) [inline easybuild/easyconfigs/p/pdf2docx/pdf2docx-0.5.8-foss-2023a-CUDA-12.1.1.eb:3]: as discussed, this easyconfig can be removed. Superseded by #20416

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen3

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Merging, thanks @WilleBell !

**Pull request merged**